src/main/scala/org/broadinstitute/dsde/workbench/sam/api/SamUserDirectives.scala

```diff
@@ -51,15 +51,16 @@ trait SamUserDirectives {
 
   def asAdminServiceUser: Directive0
 
+  val oldTermsOfServiceUrl = "app.terra.bio/#terms-of-service"
```

Comment:
maybe we want to make this configurable?

Reply:
I don't think so. We're eventually going to get rid of this, but it still needs to be here for now since the current endpoints require this to be present in the body to accept the ToS.